pysteps/extrapolation/semilagrangian.py

```diff
@@ -173,7 +173,7 @@ def extrapolate(
 
     if xy_coords is None:
         x_values, y_values = np.meshgrid(
-            np.arange(velocity.shape[2]), np.arange(velocity.shape[1])
+            np.arange(velocity.shape[2]), np.arange(velocity.shape[1]), copy=False
```

Comment:
I can see how this provides better performance, but are we 100% sure that there are no side effects?

Reply:
perhaps @pulkkins as the original author of this code can jump in here and give us a feedback on whether those copies were intentional or not?

Reply:
I think not, since the `np.arange(...)` creates a new array, and it's this new array that is no longer copied. When copy=True, the reference to the brand-new array at some point goes out of scope and the memory will be garbage collected anyway. I think this just avoids unnecessarily creating and throwing away an intermediate array.